papermill/execute.py

```diff
@@ -104,6 +109,8 @@ def execute_notebook(
             )
 
         nb = prepare_notebook_metadata(nb, input_path, output_path, report_mode)
+        if remove_tagged_cells is not None:
```

Comment:
```suggestion
        if remove_tagged_cells:
```
should be enough – the empty string isn't a valid tag, right..?